data/schema/schema_v1.json

```diff
@@ -76,7 +76,8 @@
             "type": "string"
           },
           "meta": {
-            "type": "string"
+            "type": "string",
+            "description": "Metadata item which will take a particular action in a survey"
```

Comment:
This should be "Metadata provided by the calling service. This will vary between surveys."

Reply:
Done